pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2732,4 +2732,16 @@ self: super: {
 
   # The hackage source is somehow missing a file present in the repo (tests/ListStat.hs).
   sym = dontCheck super.sym;
+
+  # Doesn't work with GHC 9.6 without this patch
+  # https://github.com/kowainik/typerep-map/pull/128
+  typerep-map = overrideSrc rec {
+    src = pkgs.fetchFromGitHub {
+      owner = "alaendle";
+      repo = "typerep-map";
+      rev = "444d377fac1d0941721233737ee65b631ea3d056";
+      sha256 = "sha256-BU/BvOI920QTW9MCfQyz+8zTRa0Ayx8i6R3matzRAIs";
+    };
+  } super.typerep-map;
```

Comment:
Is it possible to just jailbreak `typerep-map`?  That way we wouldn't have to just remember to remove this source override.

Reply:
Looks like it builds with just the jailbreak 👍  Just pushed the changes and rebased the PR